### PR TITLE
cleanup copied tokens

### PR DESCRIPTION
**Pull request by marcmengel**
we make a copy of our token because the submission monkeys with it, but
we weren't cleaning it up...

### CODE REVIEW

File: lib/utils.py [hunk 1]
```diff
@@ -68,6 +69,11 @@ def cleanup(varg: Dict[str, Any]) -> None:
             if entry.name.startswith("js_") and time.time() - sb.st_mtime > 604800:
                 cleandir(entry.name, verbose=varg["verbose"])
 
+    # if our BEARER_TOKEN_FILE ends with our pid, then we copied it and should
+    # clean it up...
+    if token_mods.is_copied_token(os.environ["BEARER_TOKEN_FILE"]):
```
Comment (shreyb) [inline lib/utils.py:74]: Can we also add a verbose clause here, where if `varg["verbose"] > 0`, we print that we cleaned up that file?

**Review by shreyb — CHANGES_REQUESTED** [review]
Looks good.  I have one tiny request.

File: lib/utils.py (hunk 1 above)
Comment (marcmengel) [reply lib/utils.py:74]: Done.

Comment (shreyb) [reply lib/utils.py:74]: Did you push that commit to Github?  The PR looks the same as before.

**Comment by marcmengel** [comment]
Sorry, @shreyb  I *thought* I pushed it...  too many balls in the air this week... sigh.

**Pull request merged**